rts/motoko-rts/src/text.rs

```diff
@@ -100,9 +100,8 @@ pub unsafe fn text_concat<M: Memory>(mem: &mut M, s1: Value, s2: Value) -> Value
     }
 
     // Create concat node
-    let r = mem.alloc_words(size_of::<Concat>());
+    let r = mem.alloc_words(size_of::<Concat>(), TAG_CONCAT);
     let r_concat = r.get_ptr() as *mut Concat;
```

Comment:
Now that you have the tag field initialized you could use `as_concat` here.

Reply:
I will `grep` for other such occurrences too.

Reply:
0cdd247!